recipe-server/normandy/settings.py

```diff
@@ -277,6 +282,9 @@ class Development(Base):
 
     API_CACHE_TIME = values.IntegerValue(0)
 
+    SWAGGER_SETTINGS = Base.SWAGGER_SETTINGS
```

Comment:
Did you mean `Core.SWAGGER_SETTINGS`? Also, I don't think you need this anyways, since it is a subclass.